usecases/no2-monitoring/index.md

```diff
@@ -0,0 +1,384 @@
+
+For more details please [visit the homepage of the R Shiny app](https://github.com/Open-EO/r4openeo-usecases/tree/main/uc3-s5p-dashboard).
+
+## Basic NO₂ analysis in Python, R and JavaScript
+
+::: warning Attention
+This tutorial assumes you have completed the Getting Started guides and are connected and logged in to openEO Platform.
+Your connection object should be stored in a variable named `connection`.
+:::
+
+openEO Platform has multiple collections that offer Sentinel-5P data, e.g. for NO₂:
+- [TERRASCOPE_S5P_L3_NO2_TD_V1](https://openeo.cloud/data-collections/view/?id=TERRASCOPE_S5P_L3_NO2_TD_V1) - daily, hosted by VITO, pre-processed to remove clouds etc.
+- [TERRASCOPE_S5P_L3_NO2_TD_V1](https://openeo.cloud/data-collections/view/?id=TERRASCOPE_S5P_L3_NO2_TM_V1) - monthly, hosted by VITO, pre-processed to remove clouds etc.
+- [TERRASCOPE_S5P_L3_NO2_TD_V1](https://openeo.cloud/data-collections/view/?id=TERRASCOPE_S5P_L3_NO2_TY_V1) - yearly, hosted by VITO, pre-processed to remove clouds etc.
+- [SENTINEL_5P_L2](https://openeo.cloud/data-collections/view/?id=SENTINEL_5P_L2) - Level 2 data, hosted by Sentinel Hub
+
+`SENTINEL_5P_L2` also contains additional data such as CO, O₂, SO₂ and you can also experiment with those. CO is also available on VITO in Collections such as `TERRASCOPE_S5P_L3_CO_TD_V1`.
+
+In this example we'll use the daily composites from the `TERRASCOPE_S5P_L3_NO2_TD_V1` collection, which is available starting from end of April 2018.
+
+### 1. Load a data cube
+
+First of all, we need to load the data into a datacube. We set the temporal extent to the year 2019 and choose spatial extent, here an area over Münster, Germany.
+
+<CodeSwitcher :languages="{py: 'Python', js: 'JavaScript', r: 'R'}">
+<template v-slot:py>
+
+```python
+year = 2019
+extent = { # Münster
+    "type": "Polygon",
+    "coordinates": [[
+        [7.737228350528245,51.86687168604513],
+        [7.507741544165615,51.86687168604513],
+        [7.507741544165615,52.05013100121914],
+        [7.737228350528245,52.05013100121914],
+        [7.737228350528245,51.86687168604513]
+    ]]
+}
+datacube = connection.load_collection("TERRASCOPE_S5P_L3_NO2_TD_V1", spatial_extent = extent, temporal_extent = [f"{year}-01-01", f"{year}-12-31"])
+```
+
+</template>
+<template v-slot:js>
+
+```js
+let builder = await connection.buildProcess();
+
+let year = 2019;
+let extent = { // Münster
+    "type": "Polygon",
+    "coordinates": [[
+        [7.737228350528245,51.86687168604513],
+        [7.507741544165615,51.86687168604513],
+        [7.507741544165615,52.05013100121914],
+        [7.737228350528245,52.05013100121914],
+        [7.737228350528245,51.86687168604513]
+    ]]
+}
+let datacube = builder.load_collection("TERRASCOPE_S5P_L3_NO2_TD_V1", extent, [`${year}-01-01`, `${year}-12-31`]);
+```
+
+</template>
+<template v-slot:r>
+
+```r
+p = processes()
+
+year = "2019"
+extent = list( # Münster
```

Comment:
This could be done more elegant. by passing an `sf` object

Reply:
Sure, but how? I've never works with sf

Reply:
on it

Reply:
This will give the square bounding box and uses the `bbox` class. It's more R-ish. The backend will anyway return the square bounding box right?
```r
extent = st_bbox(c(xmin = 5.9521131313239675, xmax = 9.893643204421315, ymax = 51.78636868072434, ymin = 50.52897921775278), crs = st_crs(4326))
```

Reply:
Thanks, but I need a geometry because I can't pass a bbox in aggregate_spatial below.

Reply:
Ok I could also pass the code for that... but I think it's from behind through the shoulder. `sf` makes sense then when loading objects. Creating by hand results in more or less the same amount of code you are using.